### ISSUE TITLE
托多：添加标题

### ISSUE THREAD

**Issue by zhang652** [koalaman/shellcheck#2268]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/your/shebang
Your script
```

#### Here's what shellcheck currently says:
```
TODO: Describe current output
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by brother** [comment]
Title is "Todo: Add title" according to google. 
https://translate.google.com/?sl=auto&tl=en&text=%E6%89%98%E5%A4%9A%EF%BC%9A%E6%B7%BB%E5%8A%A0%E6%A0%87%E9%A2%98&op=translate

**Comment by koalaman** [comment]
@brother Thanks! I wish I had much more time to keep on top of these things. I've offered you GH access to manage any such issues if you want it

**Comment by brother** [comment]
I'll keep going here in the tracker, haskell has never been my home really =)
